src/Microsoft.Health.SqlServer/RetryableInitializationOperation.cs

```diff
@@ -40,6 +48,7 @@ public RetryableInitializationOperation(Func<Task> operation)
         /// synchronization so only one running task will exist at a time.
         /// </summary>
         /// <returns>A task representing the completion of the initialization operation.</returns>
+        [SuppressMessage("Maintainability", "CA1508:Avoid dead conditional code", Justification = "Task reference may be updated by another thread.")]
```

Comment:
Analyzer erroneously thinks checking the `_task` reference after the semaphore is unnecessary